Problems with compile.sh

I have tried several times and with different versions to compile my image for the bananapi r2 device, everything has always gone well, now after the 20.05 release it seems that it is no longer able to compile my image.

![20 02](https://user-images.githubusercontent.com/43168947/88260422-a14c2600-ccc4-11ea-9bf1-07bc619a078b.png)

With version 20.02 I have the problem that 'buster' doesn't work, and that's fine, it fits.

![20 05](https://user-images.githubusercontent.com/43168947/88260507-cf316a80-ccc4-11ea-99f7-be222dc07a3d.png)

However, the problem also exists with version 20.05.
It does not give the same error but crashes during compilation to the following file.


As for the nightly or the version that I download directly from the git via the 'git clone https://github.com/armbian/build.git' command, I get the following problem.

The image is compiled, but once inserted on the micro SD as I have always done, the device does not start and shows no signs of life.
![photo_2020-07-23_09-30-45](https://user-images.githubusercontent.com/43168947/88261785-3e0fc300-ccc7-11ea-9cc9-c9725956fdb8.jpg)


The command that I have always used in all compilations, even when everything worked, is the following: "sudo ./compile.sh BOARD ="bananapir2" BRANCH ="legacy" RELEASE ="buster" BOARD_NAME ="Banana Pi R2" BOARDFAMILY ="mt7623" BOOTCONFIG ="mt7623n_bpir2_defconfig" BUILD_DESKTOP ="no" KERNEL_TARGET ="legacy" KERNEL_ONLY ="no" KERNEL_CONFIGURE ="no" KERNEL_KEEP_CONFIG ="no" BUILD_MINIMAL="yes" EXTERNAL="no" BSPFREEZE="no" INSTALL_HEADERS="no" EXTERNAL_NEW="no" CREATE_PATCHES="no" CLEAN_LEVEL="make,debs,images,alldebs,cache,oldcache,sources,extras"

Could you help me? Many thanks in advance!

I have the same problem.

**Snippet with error messages**:
```
cc1: error: unrecognized -mtune target: native
cc1: note: valid arguments are: arm2 arm250 arm3 arm6 arm60 arm600 arm610 arm620 arm7 arm7d arm7di arm70 arm700 arm700i arm710 arm720 arm710c arm7100 arm7500 arm7500fe arm7m arm7dm arm7dmi arm8 arm810 strongarm strongarm110 strongarm1100 strongarm1110 fa526 fa626 arm7tdmi arm7tdmi-s arm710t arm720t arm740t arm9 arm9tdmi arm920 arm920t arm922t arm940t ep9312 arm10tdmi arm1020t arm9e arm946e-s arm966e-s arm968e-s arm10e arm1020e arm1022e xscale iwmmxt iwmmxt2 fa606te fa626te fmp626 fa726te arm926ej-s arm1026ej-s arm1136j-s arm1136jf-s arm1176jz-s arm1176jzf-s mpcorenovfp mpcore arm1156t2-s arm1156t2f-s cortex-m1 cortex-m0 cortex-m0plus cortex-m1.small-multiply cortex-m0.small-multiply cortex-m0plus.small-multiply generic-armv7-a cortex-a5 cortex-a7 cortex-a8 cortex-a9 cortex-a12 cortex-a15 cortex-a17 cortex-r4 cortex-r4f cortex-r5 cortex-r7 cortex-r8 cortex-m7 cortex-m4 cortex-m3 marvell-pj4 cortex-a15.cortex-a7 cortex-a17.cortex-a7 cortex-a32 cortex-a35 cortex-a53 cortex-a57 cortex-a72 cortex-a73 exynos-m1 xgene1 cortex-a57.cortex-a53 cortex-a72.cortex-a53 cortex-a73.cortex-a35 cortex-a73.cortex-a53 cortex-a55 cortex-a75 cortex-a75.cortex-a55 cortex-m23 cortex-m33 cortex-r52
cc1: error: unrecognized -mtune target: native
cc1: note: valid arguments are: arm2 arm250 arm3 arm6 arm60 arm600 arm610 arm620 arm7 arm7d arm7di arm70 arm700 arm700i arm710 arm720 arm710c arm7100 arm7500 arm7500fe arm7m arm7dm arm7dmi arm8 arm810 strongarm strongarm110 strongarm1100 strongarm1110 fa526 fa626 arm7tdmi arm7tdmi-s arm710t arm720t arm740t arm9 arm9tdmi arm920 arm920t arm922t arm940t ep9312 arm10tdmi arm1020t arm9e arm946e-s arm966e-s arm968e-s arm10e arm1020e arm1022e xscale iwmmxt iwmmxt2 fa606te fa626te fmp626 fa726te arm926ej-s arm1026ej-s arm1136j-s arm1136jf-s arm1176jz-s arm1176jzf-s mpcorenovfp mpcore arm1156t2-s arm1156t2f-s cortex-m1 cortex-m0 cortex-m0plus cortex-m1.small-multiply cortex-m0.small-multiply cortex-m0plus.small-multiply generic-armv7-a cortex-a5 cortex-a7 cortex-a8 cortex-a9 cortex-a12 cortex-a15 cortex-a17 cortex-r4 cortex-r4f cortex-r5 cortex-r7 cortex-r8 cortex-m7 cortex-m4 cortex-m3 marvell-pj4 cortex-a15.cortex-a7 cortex-a17.cortex-a7 cortex-a32 cortex-a35 cortex-a53 cortex-a57 cortex-a72 cortex-a73 exynos-m1 xgene1 cortex-a57.cortex-a53 cortex-a72.cortex-a53 cortex-a73.cortex-a35 cortex-a73.cortex-a53 cortex-a55 cortex-a75 cortex-a75.cortex-a55 cortex-m23 cortex-m33 cortex-r52
make[4]: *** [scripts/Makefile.build:266: drivers/net/wireless/rtl8812au/core/rtw_security.o] Error 1
make[4]: *** Waiting for unfinished jobs....
make[4]: *** [scripts/Makefile.build:266: drivers/net/wireless/rtl8812au/core/rtw_cmd.o] Error 1
make[3]: *** [scripts/Makefile.build:500: drivers/net/wireless/rtl8812au] Error 2
make[3]: *** Waiting for unfinished jobs....
```

--- a/Makefile
+++ b/Makefile
@@ -1,5 +1,5 @@
 EXTRA_CFLAGS += $(USER_EXTRA_CFLAGS) -fno-pie
-EXTRA_CFLAGS += -O3 -mtune=native
+EXTRA_CFLAGS += -O3
 EXTRA_CFLAGS += -Wno-unused-variable
 #EXTRA_CFLAGS += -Wno-unused-value
 EXTRA_CFLAGS += -Wno-unused-label

